CHANGELOG.md

```diff
@@ -3,6 +3,42 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.3
+<!-- generated comparing v5.17.2..master -->
+_Sep 15, 2022_
+
+We'd like to offer a big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 📝 Fix lost characters when typing into edit fields (#5646) @m4theushw
```

Comment:
For this kind of description, I would mention explicitly the datagrid because the exact same sentence could make sense for the pickers

Reply:
```suggestion
- 📝 Fix lost characters when typing into fields in the DataGrid(#5646) @m4theushw
```